execution_engine_testing/tests/src/test/regression/ee_1152.rs

```diff
@@ -107,6 +107,7 @@ fn should_run_ee_1152_regression_test() {
             auction::ARG_DELEGATOR => DELEGATOR_1.clone(),
             auction::ARG_VALIDATOR => VALIDATOR_1.clone(),
             auction::ARG_AMOUNT => U512::from(DELEGATE_AMOUNT),
+            auction::ARG_NEW_VALIDATOR => Option::<PublicKey>::None
```

Comment:
if we have to update all callsites, doesn't it mean users will also have to do that? Either upgrade version of the SDK or change the code manually themselve?

Reply:
And it definitely means that if they don't do that, deploys produced by their SDKs/code will be incompatible with Ledger.

Reply:
No, this is incorrect, undelegate takes only three args: `validator`, `delegator` and `amount`. No one should need to update their SDKs to call `undelegate`. The test works because the extra argument passed here gets ignored, but thank you for pointing it out

Reply:
Does it mean it can be safely removed from the list of arguments to `undelegate` endpoint? Meaning, all the regression tests.

Reply:
Aye, I see that its still using them in places, will remove it from the list of arguments

Reply:
Removed unnecessary argument